exonum/src/runtime/rust/interfaces.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Important interservice communication interfaces.
+
+use crate::runtime::{dispatcher::Error as DispatcherError, MethodId};
+
+use super::{ExecutionError, Interface, TransactionContext};
+
+pub const INITIALIZE_METHOD_ID: MethodId = 0;
+
+/// This trait describes a service interface to pass initial configuration parameters into
```

Comment:
`This trait` is completely redundant. I suggest:
`Describes a service able to accept initial configuration parameters for the new service instance.` @Maria-Nosyk can help with better wording

Reply:
I agree that we can eliminate "This trait". As for the rest, I like the old variant more.